src/content/about-the-data/statutory-authority.mdx

```diff
@@ -1,5 +1,3 @@
 The primary statutory authority for USAspending.gov reporting and
 related data collections is the
  <a target="_blank" rel="noopener noreferrer" href="https://uscode.house.gov/view.xhtml?path=/prelim@title31/subtitle5/chapter61&amp;edition=prelim">Federal Funding Accountability and Transparency Act of 2006 (FFATA), as amended by the Digital Accountability and Transparency Act of 2014 (DATA Act) and the Congressional Budget Justification Transparency Act of 2021</a>.
-These laws are codified in
```

Comment:
This deleted copy and link needs to be restored.  The link in the first sentence needs to be removed.

Reply:
Hey, just clarifying.  The text in the first sentence should remain here.  Just remove the hyperlink from the text.  The first sentence should be - "The primary statutory authority for USAspending.gov reporting and
related data collections is the Federal Funding Accountability and Transparency Act of 2006 (FFATA), as amended by the Digital Accountability and Transparency Act of 2014 (DATA Act) and the Congressional Budget Justification Transparency Act of 2021"